packages/plugins/graphql/server/src/services/content-api/policy.ts

```diff
@@ -9,7 +9,7 @@ const getPoliciesConfig = propOr([], 'policies');
 
 const createPoliciesMiddleware = (resolverConfig: any, { strapi }: { strapi: Core.Strapi }) => {
   const resolverPolicies = getPoliciesConfig(resolverConfig);
-  const policies = policyUtils.resolve(resolverPolicies, {});
+  const policies = strapi.get('policies').resolve(resolverPolicies, {});
```

Comment:
Don't you need to remove the import too? Or is it used elsewhere?

Reply:
used for the policyContext creation which is not part of the registry